### PR TITLE
[Workers] Add 2022-04-22 update to runtime changelogs

### PR DESCRIPTION
**Pull request by KianNH**
https://community.cloudflare.com/t/2022-04-22-workers-runtime-release-notes/378427

### CODE REVIEW

File: content/workers/platform/changelog.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ title: Changelog
 
 # Changelog
 
+## 2022-04-22
+
+- Major V8 update: 9.9 → 10.0
```
Comment (deadlypants1973) [inline content/workers/platform/changelog.md]:
```suggestion
- Major V8 update: 9.9 → 10.0.
```

**Review by deadlypants1973 — APPROVED** [review]

**Comment by KianNH** [comment]
Added 2022-04-29 changelogs from https://community.cloudflare.com/t/2022-4-29-workers-runtime-release-notes/380604

File: content/workers/platform/changelog.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ title: Changelog
 
 # Changelog
 
+## 2022-04-29
+
+
+- Minor V8 update: 10.0 → 10.1.
+- R2 public beta bindings are the default regardless of compat date or flags. Internal beta bindings customers should transition to public beta bindings as soon as possible. A back compat flag is available if this isn’t immediately possible. After some lag, new scripts carrying the r2_public_beta_bindings compatibility flag will stop accepting to be published until that flag is removed.[Workers] Add 2022-04-22 update to runtime changelogs
```
Comment (deadlypants1973) [inline content/workers/platform/changelog.md]:
```suggestion
- R2 public beta bindings are the default regardless of compat date or flags. Internal beta bindings customers should transition to public beta bindings as soon as possible. A back compatibility flag is available if this is not immediately possible. After some lag, new scripts carrying the `r2_public_beta_bindings` compatibility flag will stop accepting to be published until that flag is removed.
```

**Comment by KianNH** [comment]
Squashed code review suggestion (and again due to spacing issue)

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**